src/app/static/src/components/parameters/EditParameters.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+<div>
+  <modal :open="open">
+    <dynamic-form
+      v-if="open"
+      ref="form"
+      v-model="formMeta"
+      :include-submit-button="false"
+      @submit="formSubmitted"
+    ></dynamic-form>
+    <template v-slot:footer>
+      <button class="btn btn-action"
+              @click="updateParameters">OK</button>
+      <button class="btn btn-secondary"
+              @click="cancel">Cancel</button>
+    </template>
+  </modal>
+</div>
+</template>
+
+<script lang="ts">
+import {
+    computed,
+    defineComponent,
+    Ref,
+    ref
+} from "@vue/composition-api";
+import { DynamicForm, DynamicFormMeta, DynamicFormData } from "@reside-ic/vue-dynamic-form";
+import Modal from "@/components/Modal.vue";
+import { ParameterGroupMetadata } from "@/types";
+
+interface Props {
+    open: boolean;
+    paramGroup: ParameterGroupMetadata
+}
+
+export default defineComponent({
+    name: "EditParameters",
+    props: {
+        open: Boolean,
+        paramGroup: Object
+    },
+    components: {
+        Modal,
+        DynamicForm
+    },
+    setup(props: Props, context) {
```

Comment:
I've not seen this syntax before.

Reply:
Welcome to [the future](https://v3.vuejs.org/guide/composition-api-introduction.html) :)